backend/src/cache/mutation.go

```diff
@@ -76,3 +86,46 @@ func mutatePodIfCached(req *v1beta1.AdmissionRequest) ([]patchOperation, error)
 
 	return patches, nil
 }
+
+func isValidPod(pod *corev1.Pod) bool {
+	annotations := pod.ObjectMeta.Annotations
+	if !isKFPArgoPod(&annotations) {
+		log.Printf("This pod is not created by KFP.")
```

Comment:
I'm not sure this message is helpful.
If we decide to leave this message, then we should probably include the pod name.